File: 250326/정렬된 수 위치 알아내기/indices-of-sorted-array.py
n = int(input())
sequence = list(map(int, input().split()))

# Please write your code here.

answer = []
List = sequence.copy()
List.sort()


for i in range(n):
    idx = List.index(sequence[i])
    answer.append(idx+1)
    List.pop(idx)
    List.insert(idx,0)


for i in answer:
    print(i,end=" ")
    

File: 250630/정렬된 수 위치 알아내기/indices-of-sorted-array.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');

const n = Number(input[0]);
const arr = input[1].split(' ').map(Number);

// Please Write your code here.

class answer{
    constructor(num,idx){
        this.num = num;
        this.idx = idx;
    }
}

const arr1 = arr;
const arr2 = []
console.log(arr1);
for(let i = 0 ; i < n ; i++){
    arr2.push(new answer(arr[i],i+1));
}

